server.go

```diff
@@ -63,10 +66,25 @@ func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 			s.Logger.Printf("Errored %v", err)
 		}
 
+		layoutHtml := []byte("")
+		contentHtml := []byte("")
+		isLayout := true
+
 		for _, result := range results {
```

Comment:
```suggestion
		for _, result := range results[1:] {
```

I think this will let us loop over only non-layout results